CPAC/pipeline/cpac_pipeline.py

```diff
@@ -1250,329 +1308,368 @@ def build_workflow(subject_id, sub_dict, cfg, pipeline_name=None,
         if rpool.check_rpool(func):
-            apply_func_warp['T1'] = False
-
-    target_space_nuis = cfg.nuisance_corrections['2-nuisance_regression'][
-        'space']
-    target_space_alff = cfg.amplitude_low_frequency_fluctuation['target_space']
-    target_space_reho = cfg.regional_homogeneity['target_space']
-
-    if apply_func_warp['T1']:
-
-        ts_to_T1template_block = [apply_phasediff_to_timeseries_separately,
-                                  apply_blip_to_timeseries_separately,
-                                  warp_timeseries_to_T1template,
-                                  warp_timeseries_to_T1template_dcan_nhp]
+            apply_func_warp["T1"] = False
+
+    target_space_nuis = cfg.nuisance_corrections["2-nuisance_regression"]["space"]
+    target_space_alff = cfg.amplitude_low_frequency_fluctuation["target_space"]
+    target_space_reho = cfg.regional_homogeneity["target_space"]
+
+    if apply_func_warp["T1"]:
+        ts_to_T1template_block = [
+            apply_phasediff_to_timeseries_separately,
+            apply_blip_to_timeseries_separately,
+            warp_timeseries_to_T1template,
+            warp_timeseries_to_T1template_dcan_nhp,
+        ]
 
-        if 'Template' in target_space_alff or 'Template' in target_space_reho:
+        if "Template" in target_space_alff or "Template" in target_space_reho:
             ts_to_T1template_block += [warp_timeseries_to_T1template_deriv]
 
-        if cfg.nuisance_corrections['2-nuisance_regression']['create_regressors']:
-            ts_to_T1template_block += [(warp_timeseries_to_T1template_abcd, ('desc-preproc_bold', 'bold'))]
+        if cfg.nuisance_corrections["2-nuisance_regression"]["create_regressors"]:
+            ts_to_T1template_block += [
+                (warp_timeseries_to_T1template_abcd, ("desc-preproc_bold", "bold"))
+            ]
             ts_to_T1template_block.append(single_step_resample_timeseries_to_T1template)
         else:
             ts_to_T1template_block.append(warp_timeseries_to_T1template_abcd)
             ts_to_T1template_block.append(single_step_resample_timeseries_to_T1template)
 
-        pipeline_blocks += [ts_to_T1template_block,
-                            warp_sbref_to_T1template]
+        pipeline_blocks += [ts_to_T1template_block, warp_sbref_to_T1template]
 
-    if not rpool.check_rpool('space-template_desc-bold_mask'):
-        pipeline_blocks += [warp_bold_mask_to_T1template,
-                            warp_deriv_mask_to_T1template]
+    if not rpool.check_rpool("space-template_desc-bold_mask"):
+        pipeline_blocks += [warp_bold_mask_to_T1template, warp_deriv_mask_to_T1template]
 
     pipeline_blocks += [func_despike_template]
 
-    if 'Template' in target_space_alff and target_space_nuis == 'native':
-        pipeline_blocks += [warp_denoiseNofilt_to_T1template]
+    # if "Template" in target_space_alff and target_space_nuis == "native":
+    #     pipeline_blocks += [warp_denoiseNofilt_to_T1template]
```

Comment:
The import for `warp_denoiseNofilt_to_T1template` was removed in a8782f2a03f53ed243ab6a852e045dce3b9fc993 / #1889 ([diff](https://github.com/FCP-INDI/C-PAC/commit/a8782f2a03f53ed243ab6a852e045dce3b9fc993#diff-4b55c37114baa52bec4a4a7e3eb0997e58201c63c31486e768460e6b17a93525L109)).

Do we want it gone and can remove this, or should we restore the import?